server/models/Profile.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require("mongoose")
+
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true
+  },
+  lastName: {
+    type: String,
+    required: true
+  },
+  // To allow similar names
+  id: {
+    type: Number,
+    required: true
+  },
```

Comment:
Good thought, but mongoose by default adds a unique `_id` field to schemas which can be used to distinguish different profiles 👍🏼 ([docs](https://mongoosejs.com/docs/guide.html#_id))